libs/langchain/langchain/vectorstores/mongodb_atlas.py

```diff
@@ -89,6 +91,18 @@ def from_connection_string(
         embedding: Embeddings,
         **kwargs: Any,
     ) -> MongoDBAtlasVectorSearch:
+        """Construct a `MongoDB Atlas Vector Search` vector store
+        from a MongoDB connection URI.
+
+        Args:
+            connection_string: A valid MongoDB connection URI.
```

Comment:
What happens if an app connects to 2 MongoDB clusters, one which supports $vectorSearch and another that does not?

Reply:
Can a query be run on two clusters in a single command? If the query is sent to each cluster separately, moving `_use_vectorsearch` to be per-instance of a MongoDBAtlasVectorSearch should solve that issue.

Reply:
> Can a query be run on two clusters in a single command?

No.

> moving _use_vectorsearch to be an attribute of a MongoDBAtlasVectorSearch should solve that issue.

Agreed that seems like the best approach.